### PR TITLE
Tidy my plotting functions

### PR DESCRIPTION
**Pull request by lazappi**
Adjusted some of the code for plotting the best methods and summary scatter plots and described them in the visualisation README.

Also adjusted the formatting of the README a bit to make it a bit prettier on GitHub 😸.

### CODE REVIEW

**Review by LuckyMD — APPROVED** [review]
Looks good. I find most of Marta's changes in here I guess, no?

**Comment by lazappi** [comment]
Yep. I did this after Marta's PR was merged so should have all her changes.

**Comment by LuckyMD** [comment]
I guess we wait for @martaint's review and then merge? Or do you think this is fine? I took a look, but obviously not as familiar with the R plotting code as both of you.

**Comment by lazappi** [comment]
It should be fine but I wouldn't mind if @martaint could test it just to make sure the instructions make sense.

File: R/visualization/README.md
Comment (martaint) [inline R/visualization/README.md:207]: One "the" too much

File: R/visualization/README.md
Comment (martaint) [inline R/visualization/README.md:237]: here as well, two the :)

**Comment by lazappi** [comment]
Fixed the double words. @martaint were you able to run the code? If that worked fine can merge.

**Comment by martaint** [comment]
Not yet sorry I had an unexpected meeting yesterday :( gonna try it today!!

**Review by martaint — APPROVED** [review]

**Comment by lazappi** [comment]
After @martaint's testing I think this is good to merge if you are happy with that @LuckyMD?

**Comment by LuckyMD** [comment]
sounds great! Go ahead!

**Comment by LuckyMD** [comment]
Or I just did ;).

**Pull request merged**
